torch/fft/__init__.py

```diff
@@ -87,6 +87,98 @@
     tensor([0.+0.j, 1.+0.j, 2.+0.j, 3.+0.j])
 """)
 
+fftn = _add_docstr(_fft.fft_fftn, r"""
+fftn(input, s=None, dim=None, norm=None) -> Tensor
+
+Computes the N dimensional discrete Fourier transform of :attr:`input`.
+
+Note:
```

Comment:
".. note::", right?

Reply:
`Note` is one of sections supported by napoleon:
https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections